rollup.config.js

```diff
@@ -23,7 +23,8 @@ export default {
     replace({
       __BASE_DOMAIN__: production ? '' : 'http://localhost:8000',
       __GA_TRACKING_ID__: process.env.GA_TRACKING_ID,
-      __GLEAN_DICTIONARY_DOMAIN__: 'https://dictionary.telemetry.mozilla.org',
+      __GLEAN_DICTIONARY_DOMAIN__:
+        'https://deploy-preview-1163--glean-dictionary-dev.netlify.app',
```

Comment:
This needs to be changed back to `https://dictionary.telemetry.mozilla.org` before merging.